v2/pages/adminAssessment/adminAssessment.js

```diff
@@ -57,10 +57,14 @@ router.get('/', function(req, res, next) {
         res.locals.questions = result.rows;
 
         var params = {assessment_id: res.locals.assessment.id};
+	sqldb.query(sql.question_stats, params, function(err, result) {
+            if (ERR(err, next)) return;
+            res.locals.question_stats = result.rows;
+	});
```

Comment:
Closing this function here will destroy the order guarantees for when `render()` is called. In particular, `render()` might be called before this query completes.